api/rest/restapi.go

```diff
@@ -155,11 +160,8 @@ func (api *API) setupHTTP() error {
 }
 
 func (api *API) setupLibp2p(ctx context.Context) error {
-	if api.config.Libp2pListenAddr == nil && api.host == nil {
-		return nil
-	}
-
-	// Make new host. Override existing
+	// Make new host. Override any provided existing one
+	// iif we have config for a custom one,
```

Comment:
typo: `iif` and stray `,` at the end

Reply:
https://en.wikipedia.org/wiki/If_and_only_if

Reply:
doesn' t make much sense  anyways

Reply:
iff is fine, iif is not 👍